changelogs/fragments/1455-rhn-register-force.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - rhn_register - added ``force`` parameter to allow forced registering (https://github.com/ansible-collections/community.general/pull/1455).
```

https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment
```suggestion
  - rhn_register - added ``force`` parameter to allow forced registering (https://github.com/ansible-collections/community.general/issues/1454).
```
I suppose it would be more useful for users to read the issue you have created instead of this PR as that would explain why this has been implemented.

---

Agreed. I see some other fragments using the PR instead of the issue. Is the convention here essentially:
1. If this stemmed from an issue, include the issue, otherwise use the PR, or
2. Use whichever has more information
Apologies for all the questions, just want to get good practice from the start

---

I'm sure @felixfontein can help you better on your questions, but what I can tell from the docs:
> **If there is no corresponding issue**, the entry must contain a link to the PR itself.

So I guess changelog must contain a link to its issue when there is an issue present.

---

The basic idea is that the fragment always links to a relevant PR or issue. To avoid too many discusisons about it, the guidelines contain explicit rules about this, which you summed up correctly: if there is an issue, link to it: if not, link to the PR.